implementations/rs/src/handler.rs

```diff
@@ -0,0 +1,89 @@
+use ethers::types::{
+    transaction::{eip2718::TypedTransaction, eip712::TypedData},
+    TransactionRequest,
+};
+use polywrap_plugin::JSON::{from_str, to_value, Value};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+pub struct EthCallParameters {
+    transaction: TransactionRequest,
+    block_tag: String,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(untagged)]
+enum EthCallParamaterTypes {
+    Tx(TransactionRequest),
+    Tag(String),
+}
+
+//@TODO: Pass a generic to this handle params function and re-use it with different types
+pub fn handle_call_params(values: &str) -> Vec<Value> {
```

Comment:
The word "handle" is causing me some confusion here, maybe a better word might be "parse"?

Reply:
ye it makes sense will do that